conn.go

```diff
@@ -636,7 +645,7 @@ func (c *Conn) recv(ctx context.Context) error {
 		return fmt.Errorf("gocql: frame header stream is beyond call expected bounds: %d", head.stream)
 	} else if head.stream == -1 {
 		// TODO: handle cassandra event frames, we shouldnt get any currently
-		framer := newFramer(c, c, c.compressor, c.version)
+		framer := newFramer(c, c, c.compressor, c.version, c.cqlProtoExts)
```

Comment:
Please do not change Framer constructor, it makes this patch unnecessarily big. 
Add a new constructor `newFramerWithExts` or use a plain setter that needs to be called before one starts using the framer.